src/EA.Weee.RequestHandlers.Tests.Unit/AatfReturn/Internal/AatfDataAccessTests.cs

```diff
@@ -354,5 +354,37 @@ public async void DeleteAatf_DeletesAatf()
                 .Then(A.CallTo(() => genericDataAccess.Remove(returnAatf)).MustHaveHappened(Repeated.Exactly.Once))
                 .Then(A.CallTo(() => context.SaveChangesAsync()).MustHaveHappened(Repeated.Exactly.Once));
         }
+
+        [Fact]
+        public async Task GetDetails_GivenAatfId_ComplianceYearsShouldBeReturned()
```

Comment:
Do test where aatf id does not match no result should be returns

Reply:
Added